src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,489 @@
+
+-> This section can be repeated multiple times.
+
+Example:
+
+```toml
+[[events_observer]]
+endpoint = "address-to-my-local.stacks-node-api.com:3700"
+retry_count = 255
+events_keys = ["*"]
+```
+
+#### endpoint
+
+Address and port to a stacks-node-api to watch for events.
+
+Example:
+
+```toml
+endpoint = "address-to-my-local.stacks-node-api.com:3700"
+```
+
+#### retry_count
+
+Number of times to retry before failing.
+
+Example:
+
+```toml
+retry_count = 255
+```
+
+#### events_keys
+
+Event keys for which to watch.
+
+Example:
+
+```toml
+events_keys = ["*"]
+```
+
+### Section: connection_options (optional)
+
+Specifies configuration options for others connecting to the stacks node.
+
+Example:
+
+```toml
+[connection_options]
+public_ip_address = "1.2.3.4:20444"
+download_interval = 10
+walk_interval = 30
+```
+
+#### public_ip_address
+
+The advertised public IP of this stacks-node.
+
+Example:
+
+```toml
+public_ip_address = "1.2.3.4:20444"
+```
+
+#### download_interval
+
+Time (in seconds) between attempts to download blocks.
+
+Example:
+
+```toml
+download_interval = 10
+```
+
+#### walk_interval
+
+Time (in seconds) between attempts to walk the Merkle tree.
+
+Example:
+
+```toml
+walk_interval = 30
+```
+
+### Section: burnchain
+
+Example:
+
+```toml
+[burnchain]
+chain = "bitcoin"
+mode = "krypton"
+peer_host = "bitcoind.blockstack.org"
+rpc_port = 18443
+peer_port = 18444
+```
+
+This section contains configuration options pertaining to the blockchain the stacks-node binds to on the backend for proof-of-transfer (BTC).
+
+#### chain
+
+The blockchain stacks-node binds to on the backend for proof-of-transfer.
+
+Example:
+
+```toml
+chain = "bitcoin"
+```
+
+#### mode
+
+The profile of which to run stacks-node.
+
+Example:
+
+```toml
+mode = "krypton"
+```
+
+#### peer_host
+
+Host running the BTC blockchain.
+
+Example:
+
+```toml
+peer_host = "bitcoind.krypton.blockstack.org"
+```
+
+#### rpc_port
+
+peer_host's port stacks-node will connect to for RPC connections.
+
+Example:
+
+```toml
+rpc_port = 18443
+```
+
+#### peer_port
+
+peer_host's port stacks-node will connect to for P2P connections.
+
+Example:
+
+```toml
+peer_port = 18444
+```
+
+#### process_exit_at_block_height (optional)
+
+Block height of the burnchain at which the stacks-node will self-terminate. Used during the testnet phases for various testing cycles.
+
+Example:
+
+```toml
+process_exit_at_block_height = 5340
+```
+
+#### burnchain_op_tx_fee (optional)
+
+Transaction fee per burnchain operation.
+
+Example:
+
+```toml
+burnchain_op_tx_fee = 5500
+```
+
+#### burn_fee_cap (optional)
+
+Max burn fee for a transaction.
+
+Example:
+
+```toml
+burn_fee_cap = 30000
+```
+
+#### commit_anchor_block_within (optional)
+
+Sets the time period (in milliseconds) for commitments. Only used when `mode` is set to `"helium"`.
+
+Example:
+
+```toml
+commit_anchor_block_within = 10000
+```
+
+### Section: mstx_balance
```

Comment:
One thing I would add here -- these micro-STX balances define the "genesis block allocations" for the blockchain / testnet. If a user _changes_ these values, their node may be in conflict with other nodes on the network and find themselves unable to sync with other nodes (because they are in fact mining a fork).